Plugins/Flow.Launcher.Plugin.Program/Programs/UWP.cs

```diff
@@ -320,7 +321,29 @@ public Result Result(string query, IPublicAPI api)
                     ContextData = this,
                     Action = e =>
                     {
-                        Launch(api);
+                        var elevated = (
+                            e.SpecialKeyState.CtrlPressed &&
+                            e.SpecialKeyState.ShiftPressed &&
+                            !e.SpecialKeyState.AltPressed &&
+                            !e.SpecialKeyState.WinPressed
```

Comment:
because ctrl+alt+shift is not the same as ctrl+shift... in case we wanted to add a different action for ctrl+alt+shift we can with this check in place.